packages/teleport-component-generator-react/__tests__/integration/component-attrs.ts

```diff
@@ -1,16 +1,38 @@
 // @ts-ignore-next-line
 import ComponentWithValidPropsJSON from './component-with-valid-attr-prop.json'
 // @ts-ignore-next-line
-import ComponentWithRepeatPropsJSON from './component-with-repeat.json'
```

Comment:
can you remove this from the vue tests as well and get rid of the json file?

Reply:
done!